jsapp/js/components/header/organizationBadge.component.tsx

```diff
@@ -0,0 +1,14 @@
+import {useOrganizationQuery} from 'jsapp/js/account/stripe.api';
+import styles from './organizationBadge.module.scss';
+
+export default function OrganizationBadge() {
+  const orgQuery = useOrganizationQuery();
+
+  if (orgQuery.data?.is_mmo) {
```

Comment:
Can you move the display logic for this to the parent component? This should be a purely presentational component. You can import useOrganizationQuery hook in the main header component and check both whether the org is an mmo and whether it has a name. Then you can pass the name in to the badge as a prop.

Reply:
The parent component (`mainHeader`) is a class component and we decided to add TODO comment about this instead of spending time doing it now.